crates/bevy_ecs/src/system/query.rs

```diff
@@ -488,25 +772,29 @@ where
         }
     }
 
-    /// Gets the result of a single-result query.
+    /// Returns the single immutable query result.
```

Comment:
I would rephrase this (and the other single variants) as "returns a single immutable query result when there is exactly one entity matching the query". Some might interpret "the single immutable query result" as something like "query.iter().next()".